reference/services/index.md

```diff
@@ -1,50 +1,60 @@
-installable version found within Ubuntu package repository.
-
-| Service Name       | Package Name(s)                                             |
-| :----------------- | :---------------------------------------------------------- |
-| apache             | apache2                                                     |
-| apache-php         | apache2, libapache2-mod-php5                                |
-| apache-php-drupal  | apache2, libapache2-mod-php5, composer, mysql-client, drush |
-| apache-php7        | apache2, php7.0                                             |
-| apache-php7-drupal | apache2, php7.0, composer, mysql-client, drush              |
-| couchdb            | couchdb                                                     |
-| elasticsearch-2.4  | elasticsearch-2.4                                           |
-| elasticsearch-6    | elasticsearch-6                                             |
-| mariadb            | mariadb-server                                              |
-| memcached          | memcached                                                   |
-| mongodb            | mongodb-org                                                 |
-| mysql              | mysql-server                                                |
-| mysql-56           | mysql-server-5.6                                            |
-| nginx              | nginx                                                       |
-| nginx-php          | nginx, php5-fpm                                             |
-| nginx-php7         | nginx, php7.0                                               |
-| nodejs             | nodejs-5.x                                                  |
-| nodejs-5           | nodejs-5.x                                                  |
-| nodejs-6           | nodejs-6.x                                                  |
-| nodejs-8           | nodejs-8.x                                                  |
-| postfix            | postfix                                                     |
-| redis              | redis-server                                                |
-| supervisord        | supervisor                                                  |
-| ubuntu             | ubuntu-14.x (base image)                                    |
-
-If we're not able to install a service you need, a
-[self-hosted](https://tugboat.qa/enterprise) instance of Tugboat would give you
-full control over the services available to Tugboat.
+# Prebuilt Service Images
+
+Tugboat maintains several Docker images. These images are extensions of
+[official Docker images](https://docs.docker.com/docker-hub/official_repos) and
+include tools and configurations that help them work well with Tugboat. All of
+these images are available on [Docker Hub](https://hub.docker.com/u/tugboatqa/).
+The source code used to generate these images is available on
+[GitHub](https://github.com/TugboatQA/images).
```

Comment:
I don't quite know where, but this section should probably link back to the juicy details here: configuring-tugboat/#service-images

Reply:
The link is the other way around. The reference is meant to be for when you are already familiar with configuring-tugboat/#service-images, but just need to look up some detail, such as which images are available. There is already a link this direction.